crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -95,9 +95,9 @@ jQuery(document).ready(function($) {
     ]
   });
 
-  setBlockUI('submit', $('[data-blockui]'));
-  setBlockUI('click', $('body.backups, [data-blockui]'));
-  setBlockUI('click', $('installer-upgrades, [data-blockui]'));
+  setBlockUI('submit', '[data-blockui]');
+  setBlockUI('click', 'body.backups, [data-blockui]');
```

Comment:
Btw, I'm not sure about the selector: https://api.jquery.com/multiple-selector/ vs https://api.jquery.com/descendant-selector/

I was expecting the latter, not the former...